### PR TITLE
Test: project picker tests

### PR DESCRIPTION
**Pull request by ahtrotta**
Fixes: applandinc/board#90

Added tests for the project picker. Four possible states are tested: empty project, one good project, one bad project, and four projects.

### CODE REVIEW

**Review by apotterri — APPROVED** [review]
I fixed up the history for this PR. Before you merge, can you look over the changes and make sure I didn't miss anything, please?

I also tweaked the commit messages a bit. Most importantly, I added the `test` tag to show the type of the commits. You **must** include a type at the beginning of the subject line of at least one of the commits, as described [here](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format). Otherwise, semantic-release will ignore the commits, and your changes won't generate a release. (In this particular case, all the commits are marked `test`, so they also won't get generate a release, but that's ok.)

FWIW, I've found [this](https://cbea.ms/git-commit/) to be a useful guide to writing commit messages.

**Comment by ahtrotta** [comment]
Thanks for fixing up the PR history, I'll be sure to include a type at the beginning of future commits. I looked over the changes and you didn't miss anything.

I can't merge this because the UI Tests say `5 changes must be accepted as baselines.` and I don't think I have permission to do that. Do you know what I should do to proceed?

**Pull request merged**
